src/pages/App.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import NavBar from '../components/NavBar';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
 
 const App = React.createClass({
   render() {
     return (
-      <div>
-        <NavBar params={this.props.params}/>
-        <h1>Kodeklubben</h1>
-        {this.props.children}
-      </div>
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
```

Comment:
For å drive med server-side rendering, så må man legge til en userAgent. Kan ikke så mye om dette, men har følgende forslag (sikkert lurere ting å gjøre):
```
<MuiThemeProvider muiTheme={getMuiTheme(null, {userAgent: 'all'})}>
```

Se feks:
http://stackoverflow.com/questions/35481084/react-starter-kit-and-material-ui-useragent-should-be-supplied-in-the-muitheme
http://www.material-ui.com/#/get-started/server-rendering